.gitignore

```diff
@@ -36,6 +36,8 @@ packages/contracts-bedrock/deployments/anvil
 
 .devnet*
 
+.devnet-interop
+
```

Comment:
Is this not covered by the above `.devnet*` entry?

Reply:
good catch